server/heroku/logs.go

```diff
@@ -0,0 +1,27 @@
+package heroku
+
+import (
+	"net/http"
+
+	"github.com/remind101/empire"
+	streamhttp "github.com/remind101/empire/pkg/stream/http"
+	"golang.org/x/net/context"
+)
+
+type PostLogs struct {
+	*empire.Empire
+}
+
+func (h *PostLogs) ServeHTTPContext(ctx context.Context, w http.ResponseWriter, r *http.Request) error {
+	a, err := findApp(ctx, h)
+	if err != nil {
+		return err
+	}
+
+	err = h.StreamLogs(a, streamhttp.StreamingResponseWriter(w))
```

Comment:
Oh, almost forgot, we should wrap this in `streamhttp.Heartbeat` to prevent ELB from closing the connection.